rero_mef/utils.py

```diff
@@ -1243,11 +1243,18 @@ def get_mefs_endpoints():
     return mefs
 
 
-def generate(search):
+def generate(search, deleted):
     """Lagging genarator."""
     yield '['
-    for idx, record in enumerate(search.scan()):
+    idx = 0
+    for hit in search.scan():
         if idx != 0:
             yield ', '
-        yield json.dumps(record.to_dict())
+        record = hit.to_dict()
+        yield json.dumps(record)
```

Comment:
Why not `yield json.dumps(hit.to_dict())`?

Reply:
ok